certs/variables.tf

```diff
@@ -1,66 +1,81 @@
-# openvpn_server - Variables
+# Input Variables
 
-variable "ami" {
-  description = "Artifact AMI"
+## Resource tags
+variable "stack_item_label" {
+  type        = "string"
+  description = "Short form identifier for this stack. This value is used to create the 'Name' resource tag for resources created by this stack item, and also serves as a unique key for re-use."
 }
 
-variable "stack_item_fullname" {}
+variable "stack_item_fullname" {
+  type        = "string"
+  description = "Long form descriptive name for this stack item. This value is used to create the 'application' resource tag for resources created by this stack item."
+}
 
-variable "stack_item_label" {}
+## VPC parameters
 
-#
-# 0 - if instance is a standalone instance outside a VPC
-# 1 - if instance is in a vpc
-#
+###
+### 0 - if instance is a standalone instance outside a VPC
+### 1 - if instance is in a vpc
+###
 variable "in_vpc" {
-  default = 0
+  type        = "string"
+  description = "Flag for associating the cluster with a VPC."
+  default     = 1
```

Comment:
Probably, we should probably check some defaults for the other elements and remove them if we do switch this default.